src/Panel/BasicLocalePanel.php

```diff
@@ -10,7 +10,7 @@
 use Baraja\Url\Url;
 use Nette\Security\User;
 
-final class BasicPanel implements Panel
+final class BasicLocalePanel implements Panel
 {
 	public function __construct(
 		private Localization $localization,
```

Comment:
Klidně bych dal přijetí služby jen jako volitelnou závislost:

```suggestion
		private ?Localization $localization = null,
```

Pak ale bude potřeba upravit vnitřní logiku pro získávání informací o jazyce. Klidně bych v případě nedostupnosti služby předpokládal jen jazyk `en`.